integreat_cms/cms/models/feedback/imprint_page_feedback.py

```diff
@@ -1,16 +1,25 @@
+from django.db import models
 from django.urls import reverse
 from django.utils.functional import cached_property
 from django.utils.translation import ugettext_lazy as _
 
+
 from .feedback import Feedback
-from ..pages.imprint_page import ImprintPage
+from ..pages.imprint_page_translation import ImprintPageTranslation
 
 
 class ImprintPageFeedback(Feedback):
     """
     Database model representing feedback about imprint pages.
     """
 
+    imprint_page_translation = models.ForeignKey(
+        ImprintPageTranslation,
+        on_delete=models.CASCADE,
+        related_name="feedback",
+        verbose_name=_("imprint page translation"),
+    )
+
```

Comment:
```suggestion
```
I don't think it's required to add this field - in most cases, the imprint is just duplicated into all languages which means only for the default language this translation does exist - for all other languages it doesn't...